docs/DeveloperGuide.md

```diff
@@ -263,18 +264,22 @@ The activity diagram shows the general sequence of steps when a user interacts w
 <br>
 
 ### Clear person feature
-
-#### Implementation
-
 #### Design considerations
+**Aspect: Should `clear persons` also delete appointments**
+- Alternative 1 (Current choice): The `clear persons` command should also clear the appointments else there will be alot of appointments inside the application that are linked to person ids that don't exists. Causing confusion for the users.
```

Comment:
Just some minor grammatical nitpicking and spelling typos, would it be possible to rephrase it? It would be good keep it consistent with the format under general commands as well :)

- **Alternative 1 (current choice):** The `clear persons` command should also clear all appointments.
  - Pros: This prevents the case where the appointments are linked to deleted `personIds` which do not exist. Hence, this prevents confusion for users.